internal/reconciler/logpipeline/reconciler.go

```diff
@@ -188,6 +196,8 @@ func (r *Reconciler) reconcileFluentBit(ctx context.Context, pipeline *telemetry
 	}
 
 	cm := resources.MakeConfigMap(r.config.DaemonSet)
+
+	cm = updateConfigifNoDeployablePipelines(deployableLogpipelines, cm)
```

Comment:
Maybe we better extend resources.MakeConfigMap by a `includeSections` flag instead this hacky way.